Update rtd 01

johnmhoran opened the pull request.

johnmhoran committed:
Initial commit

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Modify configuration

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Create basic outline

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Replace /docs/ directory with new code

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Add requirements.txt

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Create new folder/file structure

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Work on UI Reference, How-To Guides

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Add section re platform support

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Clean up CSS

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

steven-esser commented on .gitignore:
```diff
@@ -17,6 +17,7 @@
 node_modules/
 bower_components/
 docs/build
+docs-update/build
```
Now that everything is in `docs/` I do not think we need this entry here.

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+/* change color of inline code block */
+/* span.pre {
+    color: #505050;
+} */
```
Is this block needed? I would remove it since we do not use it at all, unless there is a good reason to keep it around.

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+    font-style: normal;
+    font-family: "Courier New";
+    font-size: 12px;
+    font-size: 13px;
```
Why do we have two different font-size values here?

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+    border: 0;
+    background: #e6ffff;
+    font-size: 80%;
+    font-size: 16px;
```
Why do we have two different font-size values here?

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+    font-size: 80%;
+    font-size: 16px;
+    font-weight: 700;
+    font-weight: normal;
```
Why do we have two different font-weight values here?

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+.rst-content table.docutils td,
+.rst-content table.docutils th {
+    padding: 5px;
+    padding: 5px 10px 5px 10px;
```
Why do we have two different padding values here?

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+.rst-content pre.literal-block, .rst-content div[class^="highlight"] pre, .rst-content .linenodiv pre {
+    font-family: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",Courier,monospace;
+    font-size: 12px;
+    font-size: 13px;
```
Why do we have two different font-size values here?

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+    font-size: 12px;
+    font-size: 13px;
+    line-height: 1.4;
+    line-height: 1.5;
```
Why do we have two different line-height values here?

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+
+.rst-content pre.literal-block, .rst-content div[class^='highlight'] {
+    border: 1px solid #e1e4e5;
+    border: 0;
```
Why do we have two different border values here?

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+    border: 1px solid #e1e4e5;
+    border: 0;
+    background-color: #f6f8fa;
+    background-color: #f5f5f5;
```
Why do we have two different background-color values here?

steven-esser commented on docs/source/_static/theme_overrides.css:
```diff
@@ -0,0 +1,155 @@
+.rst-content .line-block {
+    margin-left: 0px;
+    margin-bottom: 24px;
+    margin-bottom: 0px;
```
Why do we have two different margin-bottom values here?

steven-esser commented on docs/scripts/doc8_style_check.sh:
```diff
@@ -2,4 +2,4 @@
 set -e
 # Check for Style Code Violations
-doc8 --max-line-length 100 ./docs/source --ignore D000 --quiet
+doc8 --max-line-length 100 source --ignore D000 --quiet
```
I think this change is causing the build failures. See: https://travis-ci.org/github/nexB/scancode-workbench/jobs/665056824#L270

steven-esser reviewed (changes requested):
This looks good, content-wise. 

I left a few comments, mostly on the css override file. The repetition is only there to point out all the various instances. Other than that, this looks good.

johnmhoran committed:
Clean up CSS, correct configuration

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Fix overly-long lines

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Add argument to code-block directives

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

johnmhoran committed:
Set index as master_doc

*  Otherwise Travis test fails because by default, Sphinx expects the
   master doc to be contents.rst, which can't be found because there is
   no such file.

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

steven-esser commented:
Something if off with Travis connection to PRs, but the build have passed so I am merging this.

johnmhoran commented:
@majurg Yes, I've been tracking this, the last Travis test passed but GH was showing it in progress, was writing a post to you.
